### ISSUE TITLE
Unable to use with React components

### ISSUE THREAD

**Issue by kdeloach** [codrops/ResponsiveMultiLevelMenu#29]
When this plugin is used within a React component, `onClick` events do not fire. Here is the line of code causing this problem:

```
this.$menuitems.on( 'click.dlmenu', function( event ) {

                event.stopPropagation();
```

Source: https://github.com/codrops/ResponsiveMultiLevelMenu/blob/master/js/jquery.dlmenu.js#L123

I suggest that `stopPropagation` should only be called when a submenu parent node is clicked, but not when normal links are clicked, to improve compatibility with React and other frameworks.

**Comment by rejas** [comment]
Care to make a pull request for this? Would like to add it to my repo fork at https://github.com/rejas/ResponsiveMultiLevelMenu

**Comment by kdeloach** [comment]
Yes, I'll try to create a PR for this. However, I noticed my fix broke some of the demos on the latest master branch so I'll have to resolve that first.
